### PR TITLE
Schema Markup and SEO changes

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jash <jashvpatel1@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6fe1acc2cf54b3e12320fb9177b56d3d0400c681 at: https://6068c295d977d307d9b54e9a--layer5ng.netlify.app

**Review by navendu-pottekkat — COMMENTED** [review]
Tested out the structured data and it looks good. Is there any other fields we can add here?

Screenshot from 2021-04-04 10-37-19

**Comment by Jashpatel1** [comment]
> Tested out the structured data and it looks good. Is there any other fields we can add here?
>

@leecalcote Any thoughts?

File: src/pages/index.js [hunk 1]
```diff
@@ -22,35 +22,50 @@ import ServiceMeshFocussed from "../sections/Home/Service-mesh-focussed";
...
+  const schema= {
+    "@context": "https://schema.org",
+    "@type": "Corporation",
+    "name": "Layer5",
+    "url": "https://layer5.io/",
+    "logo": "https://layer5.io/images/layer5-tag-dark-bg.png",
+    "sameAs": [
+      "https://twitter.com/layer5",
+      "https://www.linkedin.com/company/layer5/",
+      "https://github.com/layer5io"
```
Comment (leecalcote) [inline src/pages/index.js:35]: Perhaps, we might include https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA

**Review by leecalcote — APPROVED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (leecalcote) [reply src/pages/index.js:35]: (for the YouTube channel)

Comment (leecalcote) [reply src/pages/index.js:35]: @Jashpatel1, I assume this will just be another entry in the array...

Comment (navendu-pottekkat) [reply src/pages/index.js:35]: Yes, @leecalcote it should be another entry. The same could be done for the Meshery website too. I have added Wikidata entries for both and once we have the website ready, we should get a knowledge panel in no time.

Comment (leecalcote) [reply src/pages/index.js]: That is fantastic. This is great.

**Pull request merged**